Assets/PatchKit Patcher/Scripts/AppUpdater/AppRepairer.cs

```diff
@@ -18,6 +18,8 @@ public class AppRepairer
 
         public readonly AppUpdaterContext Context;
 
+        public ICheckVersionIntegrityCommand checkIntegrity;
```

Comment:
Instead of passing command please pass version integrity results, as only this thing is used later on.

Reply:
done